src/js/components/AuthorPicker.js

```
@@ -2,261 +2,129 @@
+ * @param {Function} props.onAuthorsChange Callback when authors change
+ * @param {number} props.maxAuthors Maximum number of authors (0 = unlimited)
  * @return {JSX.Element} Component to render
  */
-const AuthorPicker = ({ 
-    selectedAuthorIds = [], 
-    onChange, 
-    buttonLabel,
-    perPage = 100,
-    showAvatars = true
+const AuthorPicker = ({
+    authors = [],
```

ESLint

[error] Replace `····` with `↹`

(prettier/prettier)